source/unified-test-format/tests/Makefile

```diff
@@ -1,4 +1,4 @@
-SCHEMA=../schema-1.9.json
+SCHEMA=../schema-1.12.json
```

Comment:
I noticed this file is quite outdated. Since [DRIVERS-1767](https://jira.mongodb.org/browse/DRIVERS-1767) was implemented and a CI workflow was added to test spec files against their exact `schemaVersion`, this file only serves two purposes:

 * Validate a wide number of specs against the latest schema (not sure how useful or necessary this is)
 * Test schema validation failures for `invalid` tests, which are ignored by the CI workflow

FWIW, the Makefile is still quite a bit faster than the [`check_schema_version.sh`](https://github.com/mongodb/specifications/blob/master/.github/workflows/check_schema_version.sh) script since it invokes `ajv` once for an entire directory of files. Going forward, I think it'd be best to eliminate this file and update `check_schema_version.sh` to also process `invalid` tests and possible optimize it to invoke fewer instances of `ajv` (e.g. collect specs per schema and run a single `ajv` instance per schema).

Reply:
Tracking in [DRIVERS-2466](https://jira.mongodb.org/browse/DRIVERS-2466).